google_guest_agent/run/run_test.go

```diff
@@ -0,0 +1,165 @@
+//  Copyright 2023 Google Inc. All Rights Reserved.
+//
+//  Licensed under the Apache License, Version 2.0 (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+//  Unless required by applicable law or agreed to in writing, software
+//  distributed under the License is distributed on an "AS IS" BASIS,
+//  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+//  See the License for the specific language governing permissions and
+//  limitations under the License.
+
+package run
+
+import (
+	"context"
+	"fmt"
+	"os"
+	"os/exec"
+	"path"
+	"strings"
+	"testing"
+	"time"
+)
+
+// builds a set of data to be used in the tests
+func buildDataContent(t *testing.T) string {
+	t.Helper()
+	rootDir := path.Join(t.TempDir(), fmt.Sprintf("%d", time.Now().UnixNano()))
+	if err := exec.Command("mkdir", "-p", rootDir).Run(); err != nil {
```

Comment:
why not just use `os.MkdirAll()` instead of exec?

Reply:
Of course.

Reply:
Done. Thanks.